composer.json

```diff
@@ -7,20 +7,20 @@
     "require": {
         "php": "~7.1|^8.0",
         "myclabs/php-enum": "^1.2",
-        "phpstan/phpstan": "^0.10|^0.11|^0.12"
+        "phpstan/phpstan": "^0.10|^0.11|^0.12.34"
```

Comment:
Condition `^0.12.34` seems to be very strong. Why not `^0.12`?

Reply:
It does not work on php 8 with phpstan 0.12.33 (first phpstan release with php 8 compatibility) :(